sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/CallMethodViaReflection.scala

```diff
@@ -64,7 +64,10 @@ case class CallMethodViaReflection(children: Seq[Expression])
     if (children.size < 2) {
       DataTypeMismatch(
         errorSubClass = "WRONG_NUM_PARAMS",
-        messageParameters = Map("actualNum" -> children.length.toString))
+        messageParameters = Map(
+          "functionName" -> prettyName,
+          "expectedNum" -> "at least two",
```

Comment:
Let's pass `>= 2` instead of `at least two` to don't depend on English. When we will translate `error-classes.json` to a local language, it would be nice to avoid error messages in different languages.

Reply:
Done